src/ontodia/customization/templates/index.ts

```diff
@@ -13,25 +20,43 @@ export * from './person';
 
 export const DefaultTemplateBundle: TemplateResolver[] = [
     types => {
-        if (types.indexOf('http://www.w3.org/2000/01/rdf-schema#Class') !== -1) {
-            return BigIconTemplate;
-        } else if (types.indexOf('http://www.w3.org/2002/07/owl#Class') !== -1) {
-            return BigIconTemplate;
-        } else if (types.indexOf('http://www.w3.org/2002/07/owl#ObjectProperty') !== -1) {
-            return LeftBarTemplate;
-        } else if (types.indexOf('http://www.w3.org/2002/07/owl#DatatypeProperty') !== -1) {
-            return LeftBarTemplate;
-        } else if (types.indexOf('http://xmlns.com/foaf/0.1/Person') !== -1 ||
-            types.indexOf('http://www.wikidata.org/entity/Q5') !== -1 ) {
-            return PersonTemplate;
-        } else if (
-            types.indexOf('http://schema.org/Organization') !== -1 ||
-            types.indexOf('http://dbpedia.org/ontology/Organisation') !== -1 ||
-            types.indexOf('http://xmlns.com/foaf/0.1/Organization') !== -1
-        ) {
-            return OrganizationTemplate;
-        } else {
-            return undefined;
+        if (window.location.href == "http://localhost:10444/vowl.html") {
```

Comment:
I think it would be better to instead create separate `VowlTemplateBundle` and just import and use it in the demo in place of `DefaultTemplateBundle`.

Reply:
I understand what you mean, but I don't know how. Can you help me?
Not done